UnityProject/Assets/Scripts/Weapons/Projectiles/Behaviours/KeneticProjectileDamageInterity.cs

```diff
@@ -1,57 +1,58 @@
 ﻿using UnityEngine;
-using Weapons.Projectiles.Behaviours;
-using Weapons;
 using ScriptableObjects.Gun;
 
-/// <summary>
-/// Damages integrity on collision
-/// only for kenetic weapons
-/// </summary>
-public class KeneticProjectileDamageInterity : MonoBehaviour, IOnShoot, IOnHit
+namespace Weapons.Projectiles.Behaviours
```

Comment:
Namespace declaration goes before the class declaration. The entire class is inside the scope (brackets) of the namespace

Reply:
ah my bad, I was misreading the changes. This is solved

Reply:
it is

Reply:
namespace Weapons.Projectiles.Behaviours
{
	/// <summary>
	/// Damages integrity on collision
	/// only for kenetic weapons
	/// </summary>
	public class KeneticProjectileDamageInterity : MonoBehaviour, IOnShoot, IOnHit